_develop/connect/get-started.md

```diff
@@ -162,4 +157,24 @@ if (userSession.isSignInPending()) {
   const userData = await userSession.handlePendingSignIn();
   // your user is now logged in.
 }
-```
\ No newline at end of file
+```
+
+## Design Guidance
+
+Blockstack is valuable to users, but it can also be a barrier to those unfamiliar with Blockstack. The following guidelines serve to remedy that and help you onboard as many new users as you can. 
+
+### Delay Blockstack onboarding as long as possible
+
+People will often leave apps when things are asked of them before they experience the app. Give them a chance to try your app before you ask them to sign up with Blockstack. For example, a note taking app could let a new user write a couple of notes before prompting them to save their progress.
+
+### Provide an easy way in for new users
+
+Many new users to your app will not be familiar with Blockstack yet and will be hesitant to click a Blockstack-branded button. Provide a generic button for users that are new to your app and Blockstack. Blockstack Connect will introduce new users to Blockstack and recognize existing users.
+
+![Design Guidance Example](./docs/call-to-action-branding.png)
```

Comment:
the URL is broken

Reply:
Good catch, will fix

Reply:
@jasperjansz Can you provide an updated asset here that indicates the non-Messenger example app we want to highlight in the blog post as well?